pages/index.jsx

```diff
@@ -1,10 +1,20 @@
 var React = require('react');
 
+var Habitat = require('habitat');
+Habitat.load();
```

Comment:
I would say use this way instead? https://github.com/mozilla/id.webmaker.org/blob/develop/webpack.config.js#L42-L50